index.html

```diff
@@ -6538,6 +6538,46 @@
       <code>sender.track.kind</code> is not "audio", throw an
       <code>InvalidParameters</code> exception.</p>
     </section>
+    <section id="rtcdtmf2sender-interface-definition*">
+      <h3>Interface Definition</h3>
+      <div>
+      <pre class="idl">[ Constructor (RTCRtpSender sender)]
+interface RTCDTMFSender {
```

Comment:
The wording in https://drafts.fxtf.org/geometry/#DOMMatrix is good: "For historical reasons, Window objects must also have writable, configurable, non-enumerable properties named SVGMatrix and WebKitCSSMatrix whose value is the DOMMatrix interface object."